### PR TITLE
CEM-845-KitchenCard-Component

### PR DESCRIPTION
**Pull request by Ampatte2**
Added components and stories for Status, KitchenCard, and HeaderRow.

CEM-845-HeaderRow
CEM-845-Status
CEM-845-KitchenCard

### CODE REVIEW

File: src/Containers/KitchenCard/KitchenCard.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    cardHeight,
+    cardWidth,
+    cardMargin,
+    TimeComponent,
+}): React.ReactElement => {
+    return (
+        <Card key={index} cardWidth={cardWidth} cardMargin={cardMargin} cardHeight={cardHeight}>
+            <HeaderRow
+                type="h4"
+                label={`#${_id.slice(-4)}`}
```
Comment (ralph-dev) [inline src/Containers/KitchenCard/KitchenCard.tsx:51]: -4 is a magic number

File: src/Containers/KitchenCard/KitchenCard.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                display="space-between"
+                padding="0 5px"
+            >
+                <Status status={status}>{status}</Status>
+            </HeaderRow>
+            <Grid>
+                <Paragraph bold align="left">
+                    {order_type.replace('_', ' ').toLowerCase()}
```
Comment (ralph-dev) [inline src/Containers/KitchenCard/KitchenCard.tsx:59]: What's is _? Magic string.
ENUM_UNDERSCORE_FORMAT or something better named.

File: src/Containers/KitchenCard/KitchenCardItems.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    ${media(
+        'tablet',
+        `
+        font-size:0.5rem;
+    `,
+    )}
+`;
+const Heading = styled(H)<HeadingProps>`
+    display:block;
+    ${props=> !props.isFullName && 'overflow:hidden; white-space:nowrap; text-overflow:ellipsis'};
```
Comment (ralph-dev) [inline src/Containers/KitchenCard/KitchenCardItems.tsx:71]: Don't put all the styling on one line, one line per style.

File: src/Containers/KitchenCard/KitchenCardItems.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    
+`
+const Bar = styled.div<MainInterface & ResponsiveInterface>`
+    ${({ theme }) : string => `
+        background-color: ${theme.colors.text}20;
```
Comment (ralph-dev) [inline src/Containers/KitchenCard/KitchenCardItems.tsx:76]: @fpunny can you explain this 20 crap lol

File: src/Containers/Status.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
+import React, { memo } from 'react';
+import styled from 'styled-components';
+import { transition, flex, position } from '../Utils/Mixins';
+import { SmallText, SmallTextProps } from '../Text';
+const STATUS_COLORS = {
+    prepared: '#28af00',
```
Comment (ralph-dev) [inline src/Containers/Status.tsx:6]: These Colors are all magic numbers to me and for some reason not in the Theme Object meaning no theme support

**Comment by Ampatte2** [comment]
Suggested changes made sans the STATUS_COLORS. What direction should I take on that?

**Comment by ralph-dev** [comment]
> Suggested changes made sans the STATUS_COLORS. What direction should I take on that?

I would just add it to the THEME like 
PREPARING_COLOR
PREPARED_COLOR 
etc

**Comment by Ampatte2** [comment]
Status colors has been added to Main Theme as colors.status. Status component has been modified to use status theme.

**Pull request merged**